scripts/run-e2e.ts

```diff
@@ -134,6 +141,28 @@ const addRequiredDeps = async ({ cwd, additionalDeps }: Options) => {
   }
 };
 
+const addTypescript = async ({ cwd }: Options) => {
+  logger.info(`👮🏻 Adding typescript and tsconfig.json`);
+  try {
+    await exec(`yarn add -D typescript@latest`, { cwd });
+    const tsConfig = {
+      compilerOptions: {
+        baseUrl: '.',
+        esModuleInterop: true,
+        jsx: 'preserve',
+        skipLibCheck: true,
+        strict: true,
+      },
+      include: ['src/*'],
+    };
+    const tsConfigJsonPath = path.resolve(cwd, 'tsconfig.json');
+    await writeJSON(tsConfigJsonPath, tsConfig, { encoding: 'utf8', spaces: 2 });
+  } catch (e) {
+    logger.error(`🚨 Creating tsconfig.json failed`);
```

Comment:
Love these emojis 😍

Reply:
That's what I can see on windows 🥴
![image](https://user-images.githubusercontent.com/599163/82432164-edde6f80-9a8f-11ea-98e7-a6d32010f7bf.png)